extensions/metadata_exchange/plugin.cc

```diff
@@ -105,15 +105,16 @@ bool PluginRootContext::onConfigure(size_t size) {
   const std::string function = "declare_property";
   envoy::source::extensions::common::wasm::DeclarePropertyArguments args;
   args.set_type(envoy::source::extensions::common::wasm::WasmType::FlatBuffers);
-  args.set_span(
-      envoy::source::extensions::common::wasm::LifeSpan::DownstreamConnection);
+  args.set_span(envoy::source::extensions::common::wasm::LifeSpan::FilterChain);
   args.set_schema(::Wasm::Common::nodeInfoSchema().data(),
                   ::Wasm::Common::nodeInfoSchema().size());
   std::string in;
   args.set_name(std::string(::Wasm::Common::kUpstreamMetadataKey));
   args.SerializeToString(&in);
   proxy_call_foreign_function(function.data(), function.size(), in.data(),
                               in.size(), nullptr, nullptr);
+  args.set_span(
```

Comment:
Did you mean to have both bindings or just Filter Chain binding at the top?

Reply:
Just the upstream key to filterchain.